e2e/smoke/contracts/src/main/proto/contract.proto

```
@@ -0,0 +1,11 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+option java_package = "smoke.contracts.generated";
+
+package smoke.contracts;
```

Buf (1.72.0)

[error] Files with package "smoke.contracts" must be within a directory "smoke/contracts" relative to root but were in directory "e2e/smoke/contracts/src/main/proto".

(PACKAGE_DIRECTORY_MATCH)